### PR TITLE
Merging first - v1.0.0

### PR DESCRIPTION
**Pull request by rodoherty1**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+)
+
+addCommandAlias("ci-all",  ";+clean ;+compile ;+test ;+package")
+addCommandAlias("release", ";+publishSigned ;sonatypeReleaseAll")
+
+
+// For distribution to sonartype
+
+useGpg := false
+usePgpKeyHex("4BEF11849D8638711107EB75B76CCB046AAA0BF2")
```
Comment (tshy0931) [inline build.sbt]: is it safe to expose this key here?

File: sonatype.sbt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+developers := List(
+  Developer(
+    id="rodoherty1",
+    name="Rob O'Doherty",
+    email="opensource@paddypowerbetfair.com",
+    url=url("https://www.paddypowerbetfair.com")
+  )
+)
+
+//pomExtra := (
```
Comment (tshy0931) [inline sonatype.sbt]: still need these commented part?

Comment (rodoherty1) [reply sonatype.sbt]: Thanks - I'll remove this.

File: build.sbt (hunk 1 above)
Comment (rodoherty1) [reply build.sbt]: The credentials to use this key are stored in travis as described in [this post](https://ppb.technology/2018/04/18/automatic-deployment-of-open-source-projects-to-maven-central/)

**Pull request merged**